newIDE/app/src/EventsBasedBehaviorEditor/EventsBasedBehaviorEditorDialog.js

```diff
@@ -14,17 +18,32 @@ type Props = {|
   onRenameSharedProperty: (oldName: string, newName: string) => void,
 |};
 
+type State = {|
+  currentTab: TabName,
+|};
+
 export default class EventsBasedBehaviorEditorDialog extends React.Component<
   Props,
-  {||}
+  State
 > {
+  state = {
```

Comment:
can we make this file a functional component while we're at it?
It looks super simple